modules/fuzzy/src/fuzzy_F0_math.cpp

```diff
@@ -43,6 +43,262 @@
 
 using namespace cv;
 
+void ft::FT02D_FL_process_float(InputArray matrix, const int radius, OutputArray output)
+{
+    CV_Assert(matrix.channels() == 3);
+
+    int borderPadding = 2 * radius + 1;
+    Mat imagePadded;
+
+    copyMakeBorder(matrix, imagePadded, radius, borderPadding, radius, borderPadding, BORDER_CONSTANT, Scalar(0));
+
+    Mat channel[3];
+    split(imagePadded, channel);
+
+    uchar *im_r = channel[2].data;
+    uchar *im_g = channel[1].data;
+    uchar *im_b = channel[0].data;
+
+    int width = imagePadded.cols;
+    int height = imagePadded.rows;
+    int n_width = width / radius + 1;
+    int n_height = height / radius + 1;
+
+    float *c_r = new float[n_width * n_height];
+    float *c_g = new float[n_width * n_height];
+    float *c_b = new float[n_width * n_height];
```

Comment:
It is better to avoid manual memory management via "new/delete" or "malloc/free" calls. I didn't see "delete" calls here, so probably there is memory leak.
Please use std::vector<float> instead.